scrapy_zyte_api/responses.py

```diff
@@ -0,0 +1,52 @@
+from base64 import b64decode
+from typing import Dict, List, Optional
+
+from scrapy import Request
+from scrapy.http import Response, TextResponse
+
+
+class ZyteAPIMixin:
```

Comment:
I wonder if .replace method needs to be implemented as well. Some middlewares might use it to modify the response. I haven't checked which ones though - among built-in middlewares, it's HttpCompressionMiddleware, but there could be others. If I' not mistaken, if we don't override .replace, zyte_api_response data could be lost.

Reply:
Good catch! Updated it on https://github.com/scrapy-plugins/scrapy-zyte-api/pull/10/commits/9695880ef8d20873b06b9f6ba2da335754c54fcf